python-api/gstswitch/controller.py

```diff
@@ -394,3 +415,39 @@ def parse_preview_ports(cls, res):
         for tupl in liststr:
             preview_ports.append(int(tupl[0]))
         return preview_ports
+
+    def on_preview_port_added(self, cb):
+        if not hasattr(cb, '__call__'):
```

Comment:
callable was not available from python3 to python 3.2 and 2to3 will probably change it back to an isinstance call, but to appreciate the future (it's back in python3.3), I'll change that :)